OPHD/States/MapViewStateHelper.cpp

```diff
@@ -313,29 +313,31 @@ bool selfSustained(StructureID id)
 /** 
  * Indicates that a specified tile is out of communications range (out of range of a CC or Comm Tower).
  */
-bool outOfCommRange(Point<int>& cc_location, TileMap* tile_map, Tile* current_tile)
+bool inCommRange(NAS2D::Point<int> position)
```

Comment:
I think inCommRange is a bit easier to understand than outOfCommRange. This is a nice improvement.